packages/actionmenu/src/react/index.js

```diff
@@ -144,6 +148,7 @@ const ActionMenu = React.forwardRef((props, forwardedRef) => {
           _onDividerFocus: handleDividerFocus,
           _onItemFocus: focusItemAtIndex,
           _onMouseOver: focusItemAtIndexWithMouse,
+          _onSelect: handleSelect,
```

Comment:
the native react `select` uses `onChange`. is that something we want to mimic?

Reply:
on wait. this is the actionmenu and not the dropdown. i think onSelect makes sense 👍

Reply:
Yeah, it's a great point.  

I'm tempted to match onChange instead of introduce new verbiage (onSelect, as written thus far)

But then, as you point out, this isn't a dropdown.  We're not really "selecting" anything.  There's no concept of "a selection" to the user.  In the same way, there's no "change" to a new selection.

So, I guess both are flawed.  

Dropdown is going to utilize onSelect/onChange, whatever we choose.  So, there is a relationship there that I'll build.  So, if they matched, that might make a little sense.

Any further preferences or thoughts on your end?